tests/core/test_pointset.py

```diff
@@ -352,3 +352,30 @@ def test_rotate():
     pset = pv.PointSet(np_points)
     pset.rotate([[-1, 0, 0], [0, -1, 0], [0, 0, -1]], inplace=True)
     assert np.allclose(pset.points, [-1, -1, -1])
+
+
+@pytest.mark.parametrize(
+    ('grid_class', 'dim'),
+    [
+        (pv.ExplicitStructuredGrid, 3),
+        (pv.StructuredGrid, 0),
+        (pv.StructuredGrid, 1),
+        (pv.StructuredGrid, 2),
+        (pv.StructuredGrid, 3),
+    ],
+)
+def test_pointgrid_dimensionality(grid_class, dim):
+    dims = {0: (1, 1, 1), 1: (1, 42, 1), 2: (42, 1, 142), 3: (2, 42, 142)}[dim]
```

Comment:
suggest parametrizing dims and dim together instead of using a dict (similar to the other grid test)